modules/openid-federation-common/src/commonMain/kotlin/com/sphereon/oid/fed/common/jwt/JoseJwt.kt

```diff
@@ -0,0 +1,4 @@
+package com.sphereon.oid.fed.common.jwt
+
+expect fun sign(payload: String, opts: MutableMap<String, Any>?): String
+expect fun verify(jwt: String, key: Any, opts: MutableMap<String, Any>? = mutableMapOf()): Boolean
```

Comment:
Is "= mutableMapOf() " relevant for the expect class? I'd expect in the actual class is enough but I'm not sure. (Would have to try)

Reply:
Opts should not be mutable to being with

Reply:
Made opts immutable and removed the default value